mlflow-server-oc-deployment-template.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-server-deployment-template
+
+parameters:
+- description: The name for job
+  value: 'mlflow-server-test'
+  name: APPLICATION_NAME
+  required: true
+
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    dockerImageRepository: ${APPLICATION_NAME}
+    tags:
+    - name: latest
+    lookupPolicy:
+      local: true
+
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    resources:
+      limits:
+        memory: 2Gi
+        cpu: "1"
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${APPLICATION_NAME}:latest
+    source:
+      type: "Dockerfile"
+      dockerfile: "FROM docker.io/centos/python-36-centos7:latest\nRUN pip install mlflow"
```

Comment:
For now this is ok, but later it should be a s2i build similar to https://github.com/AICoE/openshift_kubeflow_workshop/blob/master/openshift/build_config_training.yaml

Reply:
I'll change this in the next commit